webpack.config.js

```diff
@@ -111,6 +115,7 @@ function getWebpackConfig( env, argv ) {
 				ignoreSrcPattern: /^node_modules/,
 				baseDist,
 			} ),
+			new BundleAnalyzerPlugin(),
```

Comment:
I don't think this should be added to the build process. Definitely not with the default settings, which runs a server and opens up a browser page on every build.
We can just have an npm script that runs the webpack build with `--json` (to generate the stats file), and runs `webpack-bundle-analyzer` via npx to review that stats file.

Reply:
I like the bundle analyser running in every build, because it always remember us to check what's happening with our dependencies. But yeah, it can be boring sometimes. So I'm removing it and I'll implement it in a separate script in another PR. https://github.com/Automattic/sensei/pull/4122/commits/db74740fd781d460b5fa6bf1aa8e59ca77541535

Reply:
I think it would add an overhead to the build, and I'm not sure it even works properly for a non-production bundle. But we can add it and enable only for production builds & without the browser open.

Reply:
I added it to our "final" build script only. https://github.com/Automattic/sensei/pull/4133 WDYT?